### PR TITLE
Docker compose deployment of Conductor + CouchDB

### PR DESCRIPTION
**Pull request by stevecassidy**
Simplifies the deployment of Conductor + Couchdb by using a single docker-compose.yml file configured with .env. 

Tries to clean up some environment variables and rename them to make their meaning clearer.  Aiming for not repeating any configuration information but deriving it from canonical settings in .env.  

No substantive changes to the conductor code except for those parts touching the environment variables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,64 @@
...
+The following variables relate to the runtime configuration of the application:
+
+* `DATACENTRAL_GROUP_PREFIX` - group prefix for data central authentication
+* `FAIMS_COOKIE_SECRET` - secret used to sign cookies - a random string
+* `COMMIT_VERSION` - current Conductor commit version
+
+## Running
+
+```bash
+sh ./keymanagement/makeInstanceKeys.sh
```
Comment (DoxasticFox) [inline README.md:28]:
```suggestion
./keymanagement/makeInstanceKeys.sh
```

The script is executable and has a shebang referring to Bash

**Comment by aragilar** [comment]
I think this looks ready to be merged. @Denubis I'll let you merge this so you can do the necessary things on the server to have things run.

**Comment by aragilar** [comment]
I'll fix the conflicts.

**Comment by aragilar** [comment]
I'm going to merge this, I think @Denubis is away, so the longer this stays open the more conflicts will happen.

**Pull request merged**